ocaml/typing/typemod.ml

```diff
@@ -1738,7 +1738,7 @@ and transl_signature env (sg : Parsetree.signature) =
         let (od, newenv) = type_open_descr env sod in
         mksig (Tsig_open od) env loc, [], newenv
     | Psig_include sincl ->
-        transl_include ~functor_:false ~loc:item.psig_loc env sig_acc sincl
+        transl_include ~functor_:false ~loc env sig_acc sincl
```

Comment:
Anything interesting going on here? This could either be (a) a small editing fragment or (b) the result of hours of debugging or (c) neither.

Reply:
(c) Neither -- if you look (way) up, you'll see `let loc = item.psig_loc`, so I just used the variable that was in scope :-)